src/main/java/space/obminyashka/items_exchange/repository/LocationRepository.java

```diff
@@ -2,6 +2,7 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import space.obminyashka.items_exchange.repository.model.City;
```

Comment:
I see import here but the class isn't used.

Reply:
done